packages/core/src/builder.class.test.ts

```diff
@@ -915,7 +915,145 @@ describe('flushGetContentQueue', () => {
 
     expect(builder['makeFetchApiCall']).toBeCalledTimes(1);
     expect(builder['makeFetchApiCall']).toBeCalledWith(
-      `https://cdn.builder.io/api/v3/content/${expectedModel}?omit=data.blocks&apiKey=${API_KEY}&fields=data&format=${expectedFormat}&userAttributes=%7B%22urlPath%22%3A%22%2F%22%2C%22host%22%3A%22localhost%22%2C%22device%22%3A%22desktop%22%7D&limit=10&model=%22${expectedModel}%22&entry=%22${expectedEntryId}%22&enrich=true&query.id=${expectedEntryId}`,
+      `https://cdn.builder.io/api/v3/content/${expectedModel}?omit=data.blocks&apiKey=${API_KEY}&fields=data&format=${expectedFormat}&userAttributes=%7B%22urlPath%22%3A%22%2F%22%2C%22host%22%3A%22localhost%22%2C%22device%22%3A%22desktop%22%7D&includeRefs=true&limit=10&model=%22${expectedModel}%22&entry=%22${expectedEntryId}%22&query.id=${expectedEntryId}`,
+      { headers: { Authorization: `Bearer ${AUTH_TOKEN}` } }
+    );
+  });
+});
+
+describe('get', () => {
+  const API_KEY = '25608a566fbb654ea959c1b1729e370d';
+  const MODEL = 'page';
+  const AUTH_TOKEN = '82202e99f9fb4ed1da5940f7fa191e72';
+
+  let builder: Builder;
+
+  beforeEach(() => {
+    builder = new Builder(API_KEY, undefined, undefined, false, AUTH_TOKEN, 'v3');
+    const builderSubject = new BehaviorSubject<BuilderContent[]>([]);
+    builderSubject.next = jest.fn(() => {});
+    builder.observersByKey[MODEL] = builderSubject;
+    builder['makeFetchApiCall'] = jest.fn((url: string) => {
+      return Promise.resolve({
+        json: () => {
+          return Promise.resolve({});
+        },
+      });
+    });
+  });
+
+  test('hits content url with includeRefs=false when passed in params', async () => {
+    const expectedModel = 'page';
+
+    const includeRefs = false;
+
+    builder.apiEndpoint = 'content';
+    builder.getLocation = jest.fn(() => ({
+      search: `?builder.params=includeRefs%3D${includeRefs}`,
+    }));
+
+    await builder.get(expectedModel, {});
+
+    expect(builder['makeFetchApiCall']).toBeCalledTimes(1);
+    expect(builder['makeFetchApiCall']).toBeCalledWith(
+      `https://cdn.builder.io/api/v3/content/${expectedModel}?omit=meta.componentsUsed&apiKey=${API_KEY}&userAttributes=%7B%22device%22%3A%22desktop%22%7D&includeRefs=${includeRefs}&model=%22${expectedModel}%22`,
```

Comment:
looking at this default value for `builder.get()`, i realize that `noTraverse=false` is missing and should be here.